dev/django-admin.ts

```diff
@@ -0,0 +1,1293 @@
+        args: {
+          name: "EXCLUDE",
+        },
+      },
+      {
+        name: "--format",
+        description: "Format of serialized data when reading from stdin.",
+        args: {
+          name: "FORMAT",
+        },
+      },
+      {
+        name: "--skip-checks",
+        description: "Skip system checks.",
+      },
+    ],
+    description: "Installs the named fixture(s) in the database.",
+    args: [
+      {
+        name: "fixture",
+        description: "Fixture labels.",
+        variadic: true,
+      },
+    ],
+  },
+  {
+    name: "makemessages",
+    icon: DJANGO_ICON_URL,
+    options: [
+      {
+        name: ["--locale", "-l"],
+        description:
+          "Creates or updates the message files for the given locale(s) (e.g. pt_BR). Can be used multiple times.",
+        args: {
+          name: "LOCALE",
+        },
+      },
+      {
+        name: ["--exclude", "-x"],
+        description:
+          "Locales to exclude. Default is none. Can be used multiple times.",
+        args: {
+          name: "EXCLUDE",
+        },
+      },
+      {
+        name: ["--domain", "-d"],
+        description: 'The domain of the message files (default: "django").',
+        args: {
+          name: "DOMAIN",
+        },
+      },
+      {
+        name: ["--all", "-a"],
+        description: "Updates the message files for all existing locales.",
+      },
+      {
+        name: ["--extension", "-e"],
+        description:
+          'The file extension(s) to examine (default: "html,txt,py", or "js" if the domain is "djangojs"). Separate multiple extensions with commas, or use -e multiple times.',
+        args: {
+          name: "EXTENSIONS",
+        },
+      },
+      {
+        name: ["--symlinks", "-s"],
+        description:
+          "Follows symlinks to directories when examining source code and templates for translation strings.",
+      },
+      {
+        name: ["--ignore", "-i"],
+        description:
+          "Ignore files or directories matching this glob-style pattern. Use multiple times to ignore more.",
+        args: {
+          name: "PATTERN",
+        },
+      },
+      {
+        name: "--no-default-ignore",
+        description:
+          "Don't ignore the common glob-style patterns 'CVS', '.*', '*~' and '*.pyc'.",
+      },
+      {
+        name: "--no-wrap",
+        description: "Don't break long message lines into several lines.",
+      },
+      {
+        name: "--no-location",
+        description: "Don't write '#: filename:line' lines.",
+      },
+      {
+        name: "--add-location",
+        description:
+          "[{full,file,never}] Controls '#: filename:line' lines. If the option is 'full' (the default if not given), the lines include both file name and line number. If it's 'file', the line number is omitted. If it's 'never', the lines are suppressed (same as --no-location). --add-location requires gettext 0.19 or newer.",
+      },
```

Comment:
It seems like `--add-location` takes an argument! (I've added it here as a suggestion, but the linter might complain!)

```suggestion
        name: "--add-location",
        description:
          "[{full,file,never}] Controls '#: filename:line' lines. If the option is 'full' (the default if not given), the lines include both file name and line number. If it's 'file', the line number is omitted. If it's 'never', the lines are suppressed (same as --no-location). --add-location requires gettext 0.19 or newer.",
        args: {
           name: "location",
           isOptional: true,
           default: 'full'
           suggestions: [
               { name: "full", description: "the lines include both file name and line number" },
               { name: "file", description: "the line number is omitted" },
               { name: "never", description: "the lines are suppressed (same as --no-location)"},
           ],
        },
      },
```

Reply:
ooo, great catch! ✨